avocado/core/job.py

```diff
@@ -198,7 +198,7 @@ def __exit__(self, _exc_type, _exc_value, _traceback):
 
     def __start_job_logging(self):
         # Enable test logger
-        fmt = ('%(asctime)s %(module)-16.16s L%(lineno)-.4d %('
+        fmt = ('%(asctime)s %(name)s L%(lineno)-.4d %('
```

Comment:
AFAICT we're losing the aligment of the "headers" (prefixes) of the log entries here, right?

Reply:
Yes, we are. Unless I'm missing something, I would prefer to have full namespace here instead of trying to set a fixed length (that will make us waste some columns in case of a short namespace).

I understand the usability of having those fixed, but although logs are made for humans I don't thing we should see those as UI. Usually we use ':' or '|' to separate this and make the parsers life easier. If we are providing a tool or a command to show those logs without the headers, then we could easily split by '|' here.

Reply:
OK, given that we're being more precise about where a log entry comes from, I think it's a fair trade-off.